pages/ecosystem.vue

```diff
@@ -297,9 +298,11 @@ const filteredMembers = computed<Member[]>(() => {
 const filteredMembersSorted = computed<Member[]>(() => {
   if (selectedSortingOption.value === "stars") {
     return filteredMembers.value.sort((a, b) => b.stars - a.stars);
+  } else if (selectedSortingOption.value === "name") {
+    return filteredMembers.value.sort((a, b) => a.name.localeCompare(b.name));
```

Comment:
If we keep the function that sorts the projects by name in the **hooks/ecosystem-conversion-utils.ts**, I believe that we can simply return `filteredMembers.value` here, since they would be already sorted by name, so we can avoid sorting again.

Reply:
updated [here](https://github.com/Qiskit/qiskit.org/pull/3478/commits/b64976347e6f182f05b96a45b4f65ead7d93cee2)